guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -124,7 +124,7 @@ Ensure {SmartProxy} has access to `{RepoRHEL7ServerSatelliteMaintenanceProductVe
 You can then check the upgrade progress without staying connected to the command shell continuously.
 For more information about using the screen command, see link:https://access.redhat.com/articles/5247[How do I use the screen command?] article in the _Red{nbsp}Hat Knowledge{nbsp}Base_.
 +
-If you lose connection to the command shell where the upgrade command is running you can see the logged messages in the `{installer-log-file}` file to check if the process completed successfully.
+If you lose connection to the command shell where the upgrade command is running you can see the logged messages in the `{SmartProxy}.log` file to check if the process completed successfully.
```

Comment:
This isn't correct in the upstream context. There (in the Katello context) the file is named `foreman-proxy-content.log`. I think a new attribute would be the best solution.

Note that in the pure Foreman scenario (without Katello) there is no equivalent scenario so `{installer-log-file}` is the closest, but the whole guide is not used there so you can leave it undefined.

Reply:
But this section is not rendered at all for upstream?

Reply:
@ekohl @evgeni 

It is only applicable for satellite and there are no impacts for forman or Katello docs, and as per the current update, the final text getting rendered is `capsule.log`. Do we still need to add new variable for this or this is fine?

Also, kindly confirm if the same change is applicable for:
guides/doc-Upgrading_and_Updating/topics/updating_capsule_server_to_next_minor_version.adoc

Thanks!

Reply:
> It is only applicable for satellite and there are no impacts for forman or Katello docs, and as per the current update, the final text getting rendered is `capsule.log`. Do we still need to add new variable for this or this is fine?

I'd ok with this version then

> Also, kindly confirm if the same change is applicable for:
> guides/doc-Upgrading_and_Updating/topics/updating_capsule_server_to_next_minor_version.adoc

Yes it is. Please include it in this PR.